storage/redis_config.go

```diff
@@ -9,16 +11,20 @@ const (
 )
 
 type redisConfig struct {
-	RedisHost     string `envconfig:"DEIS_LOGGER_REDIS_SERVICE_HOST" default:""`
-	RedisPort     int    `envconfig:"DEIS_LOGGER_REDIS_SERVICE_PORT" default:"6379"`
-	RedisPassword string `envconfig:"DEIS_LOGGER_REDIS_PASSWORD" default:""`
-	RedisDB       int    `envconfig:"DEIS_LOGGER_REDIS_DB" default:"0"`
+	Host                   string `envconfig:"DEIS_LOGGER_REDIS_SERVICE_HOST" default:""`
+	Port                   int    `envconfig:"DEIS_LOGGER_REDIS_SERVICE_PORT" default:"6379"`
+	Password               string `envconfig:"DEIS_LOGGER_REDIS_PASSWORD" default:""`
+	DB                     int    `envconfig:"DEIS_LOGGER_REDIS_DB" default:"0"`
+	PipelineLength         int    `envconfig:"DEIS_LOGGER_REDIS_PIPELINE_LENGTH" default:"50"`
+	PipelineTimeoutSeconds int    `envconfig:"DEIS_LOGGER_REDIS_PIPELINE_TIMEOUT_SECONDS" default:"1"`
+	PipelineTimeout        time.Duration
```

Comment:
might want to leave this out of the struct and make it a function on the struct. past versions (at least) of envconfig would try to parse this